### DISCUSSION TITLE
Goodbye GTR actual meaning?

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#243] [Q&A] [answered]
https://twitter.com/AmazfitGlobal/status/1700153516268876243

Is it a goodbye to the name or should we consider there will be not more updates/bugfixes for GTR4 for example?

**Comment by liqin-ZeppHealth** [comment, marked as answer]
hello @zdunex25 , It's just a goodbye to the name. as we know, there is a new upgrade for GTS4 just released today.

**Reply by zdunex25** [reply]
That's good to know. I read on reddit there will be no more updates.
